lib/globals.d.ts

```diff
@@ -25,7 +25,7 @@ declare namespace NodeJS {
     Command: any;
```

Comment:
Can you add a file comment to the top of this file explaining what it's for? Especially with `globals.ts` floating around, it's confusing.